src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.Logger;
```

Comment:
The import for `Logger` should be from `org.apache.logging.log4j.Logger` instead of `org.apache.logging.log4j.core.Logger`. The `core` package is typically used for internal purposes and not for general logging usage.